### DISCUSSION TITLE
Is |example a nerorg syntax?

### DISCUSSION THREAD

**Discussion by sheldonj84** [nvim-neorg/neorg#1309] [Q&A] [answered]
Looking at the documentation, I see `|example end|` through out it. Is this Neorg syntax or is it just something the author chose to use? 

I couldn't see it explained in the specification anywhere. 

Thanks

**Comment by pysan3** [comment, marked as answer]
It's valid and explained in the `standard ranged tags` section.

**Reply by sheldonj84** [reply]
Thanks, trying to search for it didn't help since there is so many examples
